content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -0,0 +1,159 @@
+---
+title: >
+  Turn Visual Studio Code into an ML experimentation platform with the DVC
+  extension
+date: 2022-06-14
+description: >
+  Today we are releasing the DVC extension, which brings a full ML
+  experimentation platform to VS Code.
+descriptionLong: >
+  Today we are releasing the DVC extension, which brings a full ML
+  experimentation platform to VS Code. Manage your data, run experiments,
+  compare metrics, and visualize plots, all from the comfort of your IDE.
+picture: 2022-06-14/header.png
+author: rob_dewit
+# commentsUrl:
+tags:
+  - VS Code
+  - DVC
+  - MLOps
+  - Release
+---
+
+Since its beta release in 2017, DVC has become an essential tool for many data
+science teams. Its data versioning capabilities, reproducible pipelines, and
+experiment tracking features are at the core of our ecosystem of open MLOps
+tools.
+
+Today we are proud to launch a new product that extends how ML teams can use
+DVC: our extension for Visual Studio Code.
+
+With this extension you get a full VS Code-native experimentation platform for
+your machine learning projects. You can control datasets, run experiments, view
+metrics, create plots, and much more. All in one place in your IDE.
+
+[Install the DVC extension from the marketplace to get started.](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
+
+# Why a VS Code extension?
```

Comment:
I still think it takes way too long to get the essence of the blog post / first demo / first video. What are the other options if we want to keep this motivation / why? part?

Reply:
I feel we could manage to get the whole section into a single paragraph and merge it with an intro

Reply:
Did another iteration, and the text is now more succinct.

I'm having a hard time reducing the `Why` section further without either removing the necessary context or making the text extremely dry. I think we need to motivate our reasoning for an extension to a reasonable degree; otherwise, people may think that we simply feel that DVC is not good enough.

Visually, I agree that it would be good to get to the first demo/video faster. I think it might be good to include a first showcase before the `Why` section. Another option would be to move the `Why` section to after the features, but I don't think that would improve the flow of the post.

Thoughts?

Reply:
I agree that it takes too long to get to a visual of the product, but I do like the piece's flow.  I think this can be solved by putting one of the gifs (probably with an experiment table) above the link to the marketplace.  Then continue on.